### PR TITLE
Plant box

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
This came to me in a dream,
As I was sleeping, the egg man showed up in his egg domain.

His eyes were EGG, his mouth was EGG.
With the sound of 1000 eggs cracking he said to me, EGG.
"W..what?"
His voiced echoed louder, EGG.

Then and there I knew, the Egg man was coming, and he is coming to collect his eggs.

Here I am waking up, shaken in eldritch fear, knowing my time is short in this world, for the egg man is coming and I most pay his toll

Egg is life, egg is love.

## Why / Balance
Remove some entity lags by giving players actual big storage for plants

## Technical details
.yml

## Media
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame,

## Breaking changes
N/A

**Changelog**
:cl: dvir01,gentlebutter
- add: Added the plant box storage, you can now buy a big storage for plants from cargo.

### CODE REVIEW

**Comment by dvir001** [comment]
This PR was changed to only contain the plant box for now

File: Resources/Prototypes/Entities/Structures/Storage/ore_box.yml [hunk 1]
```diff
@@ -54,6 +50,7 @@
           True: { visible: false }
           False: { visible: true }
   - type: Storage
+    capacity: 900 # Frontier - 525<900
```
Comment (Cheackraze) [inline Resources/Prototypes/Entities/Structures/Storage/ore_box.yml:53]: im guessing storage rework broke this

File: Resources/Prototypes/_NF/Entities/Structures/Storage/plant_box.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      visible: true
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.StorageVisuals.Open:
+        top:
+          True: { visible: false }
+          False: { visible: true }
+  - type: Storage
+    capacity: 900 # Frontier - 525<900
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Structures/Storage/plant_box.yml:53]: rip capacity

**Comment by dvir001** [comment]
Fixed

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**